### PR TITLE
docs:reporting-server: Update reporting-server documentation

### PR DESCRIPTION
**Pull request by matiasbavera**
Signed-off-by: Matias Bavera <matiasbavera@gmail.com>

## What's new
Added documentation on the reporting-server

## Self-checks

- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

## Discussion
Hi @vallq , can you give me a hand here and provide me some doubts that you had, to write it down here. Some blockers should work too, we can create a FAQ.

### CODE REVIEW

File: packages/reporting-server/README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Description
+
+This project is about a server that exposes two APIs, one for writing logs to a persistent storage and the other for generate reports. As the project's name says, the idea is to work as a report server. This report server is designed to receive data from different pods within a Kubernetes cluster. The data is received through a data collection tool called Fluentd. Because fluentD is used, this project uses a parser for that format. You can find the log formats [here](https://github.com/open-rmf/rmf-web/blob/main/packages/reporting-server/rest_server/__mocks__/raw_data.py).
+
+In the following image, we can observe how all the pods interact with the reporting server
+![reporting-server](https://user-images.githubusercontent.com/11761240/123880883-9147b800-d911-11eb-88e8-f771914089ae.png)
```
Comment (vallq) [inline packages/reporting-server/README.md:6]: Thanks for the diagram! I corrected a typo
```suggestion
![rmf-web kubernetes cluster diagram](https://user-images.githubusercontent.com/28668944/123916706-8b56e300-d9b4-11eb-990f-69e717f87b38.png)

```

Comment (vallq) [reply packages/reporting-server/README.md:6]: rmf-web kubernetes cluster diagram

**Review by vallq — CHANGES_REQUESTED** [review]
I think this line `npm run bootstrap -- packages/reporting-server` has to be updated with `lerna bootstrap --scope=reporting-server` too!

Thanks for this, @matiasbavera . I think it's a lot clearer with the diagrams and the explanation of the roles of ports 8002 and 8003.

**Comment by vallq** [comment]
image
Oh yea I don't know if this should go here but when I run `reporting_server` in my terminal, and use CTRL+C to quit, the process lingers on the terminal.
image
I have to use `kill -9 <process id>` on another terminal. It works fine for the live reload uvicorn command.

**Comment by matiasbavera** [comment]
Thanks for the comments @vallq

**Review by vallq — APPROVED** [review]
Thanks for updating the docs! I will merge this in. And we can make subsequent edits if other comments come up.

**Pull request merged**